### PR TITLE
Colbass Trace Detection added

### PR DESCRIPTION
**Pull request by dhavalveera**
#7771

### CODE REVIEW

File: src/technologies/c.json [hunk 1]
```diff
@@ -2516,6 +2516,18 @@
     "scriptSrc": "https?://www\\.hashing\\.win/scripts/min\\.js",
     "website": "https://www.coinimp.com"
   },
+  "Colbass": {
+    "cats": [
+      5
+    ],
+    "description": "Colbass is AI System that transforms text into real-human voiceovers.",
+    "dom": "audio#colbass-tts-audio",
```
Comment (nurbek91) [inline src/technologies/c.json:2524]: Please check whether it is possible to determine the technology using this trace. The less resource intensive option should be
```
"js": {
      "colbassRunning": ""
    },
```

File: src/technologies/c.json [hunk 2]
```diff
@@ -2516,6 +2516,18 @@
...
+    "icon": "colbass.png",
```
Comment (nurbek91) [inline src/technologies/c.json:2525]: 32x32
Colbass
Icon with transparent background

File: src/technologies/c.json [hunk 3]
```diff
@@ -2516,6 +2516,18 @@
...
+    "pricing": [
+      "low"
+    ],
```
Comment (nurbek91) [inline src/technologies/c.json:2528]:
```
"pricing": [
      "low",
      "onetime",
      "recurring"
    ],
```

File: src/technologies/c.json (hunk 2 above)
Comment (nurbek91) [reply src/technologies/c.json:2525]: https://github.com/wappalyzer/wappalyzer/pull/7667#issuecomment-1557672416
Here, too, an icon with a transparent background is needed, if not, could you write that the request cannot be completed

**Review by nurbek91 — CHANGES_REQUESTED** [review]
Hey mate, thanks for PR!!

File: src/technologies/c.json (hunk 2 above)
Comment (dhavalveera) [reply src/technologies/c.json:2524]: I didn't get you for the 2nd message where you've mentioned another PR

Comment (nurbek91) [reply src/technologies/c.json:2524]: ok never-mind

**Review by nurbek91 — APPROVED** [review]

**Pull request merged**